frame/support/procedural/src/pallet/parse/mod.rs

```diff
@@ -62,6 +66,25 @@ pub struct Def {
 }
 
 impl Def {
+	fn mark_benchmarking_cached_calls(storage_def: &mut StorageDef, item: &mut syn::Item) {
+		if let syn::Item::Type(typ) = item {
+			for attr in typ.attrs.as_slice() {
+				if attr.style == AttrStyle::Outer {
+					if let Some(seg) = attr.path.segments.last() {
+						if let Ok(_) = syn::parse2::<keyword::benchmarking>(seg.to_token_stream()) {
+							if let Ok(group) = syn::parse2::<Group>(attr.tokens.clone()) {
```

Comment:
I think there's a convenience macro for this in `syn` where it makes it easier to un-bracket an expression:

```rust
let content;
syn::bracketed!(content in seg);
// content will now contain tokens within the parenthesis
```

Reply:
thanks! noted for next time, though we are parsing differently now. thanks a bunch for trying to help clean this up though!